src/authTokenInterceptor.ts

```diff
@@ -179,7 +178,9 @@ export const authTokenInterceptor = ({
     ms(typeof tokenExpireFudge === 'string' ? tokenExpireFudge : `${tokenExpireFudge}s`) / 1000
   applyStorage(getStorage())
 
-  return async (requestConfig: AxiosRequestConfig): Promise<AxiosRequestConfig> => {
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any -- Waiting for a fix in axios types
+  return async (requestConfig: any): Promise<any> => {
```

Comment:
does it have to be `any`?

Reply:
As of now I wasn't able to resolve the new 'InternalAxios' types to the old ones